transactor/src/test/scala/io/mediachain/transactor/CborSerializationSpec.scala

```diff
@@ -43,12 +55,59 @@ object CborSerializationSpec extends BaseSpec {
       meta = Map("created" -> CString("the past"))
     )
 
+    val entityUpdateCell = EntityUpdateCell(
+      entity = multihashRef("foo"),
+      chain = Some(multihashRef("bar")),
+      meta = Map("favoriteColor" -> CString("Blue! No, Yellow!!!"))
```

Comment:
I looked into scalacheck generators but they don't seem to be well suited for this task. I think the ad-hoc bags of words are OK for now; we can swap them out for being backed by more sophisticated generators later on.